entwatch/ze_mapeadores_museum_test4.cfg

```diff
@@ -0,0 +1,231 @@
+    }
+    "5"
+    {
+        "name"              "Pulverizer"
+        "shortname"         "Pulverizer"
+        "color"             "{red}"
+        "buttonclass"       "func_button"
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "true"
+        "forcedrop"         "true"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "422468"
+        "mode"              "3"
+        "maxuses"           "1"
+        "cooldown"          "0"
+        "maxamount"         "8"
+    }
+    "6"
+    {
+        "name"              "Saw"
+        "shortname"         "Saw"
+        "color"             "{orange}"
+        "buttonclass"       ""
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "true"
+        "forcedrop"         "true"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "422614"
+        "mode"              "0"
+        "maxuses"           "0"
+        "cooldown"          "0"
+        "maxamount"         "8"
+    }
+    "7"
+    {
+        "name"              "Magma Push"
+        "shortname"         "Magma Push"
+        "color"             "{red}"
+        "buttonclass"       "func_button"
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "true"
+        "forcedrop"         "true"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "30745"
+        "mode"              "2"
+        "maxuses"           "0"
+        "cooldown"          "40"
+        "maxamount"         "2"
+    }
+    "8"
+    {
+        "name"              "ZM Bulldozer"
+        "shortname"         "ZM Bulldozer"
+        "color"             "{orange}"
+        "buttonclass"       ""
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "false"
+        "forcedrop"         "false"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "422989"
+        "mode"              "0"
+        "maxuses"           "0"
+        "cooldown"          "0"
+        "maxamount"         "8"
+    }
+    "9"
+    {
+        "name"              "ZM Slasher"
+        "shortname"         "ZM Slasher"
+        "color"             "{darkorange}"
+        "buttonclass"       "func_button"
+        "filtername"        ""
+        "hasfiltername"     "false"
+        "blockpickup"       "false"
+        "allowtransfer"     "false"
+        "forcedrop"         "false"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "423040"
+        "mode"              "1"
+        "maxuses"           "0"
+        "cooldown"          "8"

```

Comment:
If ZM Slasher is set to mode 1, having a cooldown other than 0 on it will do nothing, as the item will still only display a [+] in the hud and scoreboard on 1 (and any mode < 1 is auto changed to 1 as well) and won't print uses in chat.

Reply:
Yeah I'm not sure where the line is on "spammy" items. On SSS they usually get two of them if I'm thinking of the right item so they'd be spamming chat quite a bit. As for putting the cooldown, I assumed it would work like that but in case someone wanted to change the mode later they could do so without altering cooldown. Should change the mode to 2 then?

Reply:
Completely up to how spammy you think it would be. Even if you leave the mode as 1 with non-zero cooldown, it won't break anything (it just also won't do anything in game).